modules/aleph/src/Aleph/Entity/AlephMaterial.php

```diff
@@ -261,6 +262,26 @@ public function getSubLibraryCode() {
     return $this->subLibraryCode;
   }
 
+  /**
+   * Set the physical location(s) of the material.
+   *
+   * @param array $placements
+   *    Array of locations.
+   */
+  public function setPlacements($placements) {
+    $this->placements = $placements;
+  }
+
+  /**
+   * Return the physical location(s) of the material.
+   *
```

Comment:
A comment about the order would be good - is this a list of multiple placements for the material, or is it hierarchical?